src/py/flwr/server/strategy/bulyan_test.py

```diff
@@ -127,4 +127,6 @@ def test_aggregate_fit() -> None:
     if actual_aggregated:
         actual_list = parameters_to_ndarrays(actual_aggregated)
         actual = actual_list[0]
-    assert (actual == expected[0]).all()
+        assert (actual == expected[0]).all()
+    else:
+        raise ValueError("aggregate_fit function returned None")
```

Comment:
It would be better to use assertions in unit tests.

```suggestion
    assert (actual == expected[0]).all()
```